Add static top-level landing page

comzeradd opened the pull request:
* Fixes #251

At the moment this includes some static content and assets that would gradually been removed as we extend the template models.

This includes also the rendered css from sass. This can be removed once #234 is merged and we add sass as part of the static compression pipeline.

akatsoulas commented on coss/base/static/css/coss.scss:
```diff
@@ -0,0 +1,124 @@
+@import 'colors';
```
Shouldn't this file along with others that are referring to the content be part of the `home` app and not the `base`?

akatsoulas commented:
Also let's merge first PR #234 and use the compressor for the assets.

comzeradd commented:
PR Updated and rebased.

I split up some assets and css code to live under the `home` app. Most of the css should part of the base app, since it's an implementation of the overall coss ui theme, and parts of that will be overwritten per app (eg. colors).

akatsoulas commented:
I am not sure as to why this has to go into the base app. Even if this is part of the `groups and gatherings` it should live under the home app. Base app should have only the bare minimum (eg login capabilities and nothing related to the UI)

comzeradd commented:
I don't fully agree, since most of these things are not app specific (not even home app specific). But let's not waste more time on this. I moved all assets under the home app. We can revisit later if needed, when the rest of the project starts to shape.
